src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -88,6 +97,24 @@ def get_by_username(
                 return list(qs.filter(email__iexact=username))
         return []
 
+    def get_organizations(
+        self,
+        *,
+        user_id: int,
+        only_visible: bool = False,
+    ) -> List[RpcOrganizationSummary]:
+        org_ids = OrganizationMemberMapping.objects.filter(user_id=user_id).values_list(
+            "organization_id", flat=True
+        )
+        if only_visible:
+            # TODO: Combine this with OrganizationMapping query
+            # (which will probably require adding status to OrganizationMapping and backfilling)
+            orgs = Organization.objects.filter(id__in=org_ids, status=OrganizationStatus.ACTIVE)
+            return [serialize_organization(o) for o in orgs]
```

Comment:
As noted inline, we'll probably need to modify the `OrganizationMapping` schema in order to support the `only_visible` parameter. Which would also mean backfilling it and adding functionality to push updates to the outbox. The stopgap solution (reading from `Organization`) should mean that this *is* safe to push to production but *won't* let us add `stable=True` to affected tests yet.